src/content/en/fundamentals/security/csp/index.md

```diff
@@ -104,7 +110,8 @@ fonts could be enabled via `font-src https://themes.googleusercontent.com`.
 This directive applies to `<frame>`, `<iframe>`, `<embed>`, and `<applet>` tags.
 This directive can't be used in `<meta>` tags and applies only to non-HTML
 resources.
-* **`frame-src`** deprecated. Use **`child-src`** instead.
+* **`frame-src`** was deprecated in level 2, but appears not to be in level 3.
+  Exactly how this will work is still behing hashed out.
```

Comment:
We're done hashing. Item 2 of https://w3c.github.io/webappsec-csp/#changes-from-level-2 is what CSP3 is going to run with. Pinky swear.